tcms/testplans/static/testplans/js/get.js

```diff
@@ -46,6 +46,10 @@ $(document).ready(function() {
         });
     });
 
+    jsonRPC('TestRun.filter', {plan: testPlanId}, function(data) {
```

Comment:
We need only active TRs shown, last 5 only, most recent first. 

Given these constraints it may be easier to render on the backend directly.